src/webots/gui/WbSimulationView.cpp

```diff
@@ -555,7 +562,7 @@ void WbSimulationView::startVideoCapture(const QString &fileName, int codec, int
 
 void WbSimulationView::stopVideoCapture(bool canceled) {
   WbVideoRecorder::instance()->stopRecording(canceled);
-  restoreFastModeIfNecessary();
+  restoreRenderingIfNecessary();
```

Comment:
```suggestion
  restoreRenderingIfNecessary();
```
This function name is wrong.
If a video is recorded but the rendering is turned off, the rendering is turned while recording the video and the turned off again when the video recording is over.
So something like this would be a better function name
```suggestion
  restoreNoRenderingIfNecessary();
```